src/components/Discussion/Composer/docs.imports.js

```diff
@@ -66,15 +67,23 @@ export const CommentComposerPlayground = () => {
     actions: {
       openDiscussionPreferences: () => Promise.resolve({ ok: true })
     },
+    composerHints: [
+      function formattingAsteriks(text) {
+        // Math where asterix is within a word (not next to whitespace) "n*n" for example
+        const hasUnescapedAsterix = !!text.match(/[^\\*\s:]\*[^*\s:]/)
+        if (hasUnescapedAsterix) {
+          return (
+            <Label>{t('styleguide/CommentComposer/formatting/asterisk')}</Label>
```

Comment:
The * is called asterisk. The uses with the typo `asteriks` and `asterix` should probably be fixed.